### PR TITLE
fix: ts, jest and typing problems

### PR DESCRIPTION
**Pull request by PlayMa256**
Fixing some typing problems

### CODE REVIEW

File: packages/web/src/relay/createQueryRendererModern.tsx [hunk 1]
```diff
@@ -14,9 +13,13 @@ type Config = {
...
 
-export default function createQueryRenderer(
-  FragmentComponent,
-  Component,
+type FragmentComponentTypes = {
+  query?: Variables;
+}
+
+export default function createQueryRenderer<P>(
+  FragmentComponent: RelayContainer<P> & FragmentComponentTypes,
```
Comment (PlayMa256) [inline packages/web/src/relay/createQueryRendererModern.tsx]: Not sure what is the correct type here.
Ts still arguing with me about it.

File: packages/web/jest.config.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ module.exports = {
   bail: false,
   snapshotSerializers: ['jest-serializer-html'],
   displayName: 'web-app',
-  setupTestFrameworkScriptFile: require.resolve('./jest.setup.js'),
+  setupFilesAfterEnv: [require.resolve('./jest.setup.js')],
```
Comment (sibelius) [inline packages/web/jest.config.js]: nice

File: packages/web/src/relay/createQueryRendererModern.tsx (hunk 1 above)
Comment (sibelius) [reply packages/web/src/relay/createQueryRendererModern.tsx]: what is the error?

Comment (PlayMa256) [reply packages/web/src/relay/createQueryRendererModern.tsx]:
```
Type '{ query: Variables; children?: ReactNode; }' is not assignable to type 'IntrinsicAttributes & MappedFragmentProps<Pick<P, Exclude<keyof P, "relay">>> & ComponentRef & { children?: ReactNode; }'.
  Property 'query' does not exist on type 'IntrinsicAttributes & MappedFragmentProps<Pick<P, Exclude<keyof P, "relay">>> & ComponentRef & { children?: ReactNode; }'.ts(2322)
```

Comment (sibelius) [reply packages/web/src/relay/createQueryRendererModern.tsx]: each fragment will have different props

Comment (PlayMa256) [reply packages/web/src/relay/createQueryRendererModern.tsx]: so any in this scenario is reasonable?

Comment (kuttim) [reply packages/web/src/relay/createQueryRendererModern.tsx]: Any updates on passing the test? 
Talking about `Type '{ query: Variables; children?: ReactNode; }' is not assignable to type 'IntrinsicAttributes & MappedFragmentProps<Pick<P, Exclude<keyof P, "relay">>> & ComponentRef & { children?: ReactNode; }'.
  Property 'query' does not exist on type 'IntrinsicAttributes & MappedFragmentProps<Pick<P, Exclude<keyof P, "relay">>> & ComponentRef & { children?: ReactNode; }'.ts(2322)`

Comment (PlayMa256) [reply packages/web/src/relay/createQueryRendererModern.tsx]: I'm trying to fix some others on #82, but i'm having problems with jest and relay. If you want to help, go give it a try :)

Comment (kuttim) [reply packages/web/src/relay/createQueryRendererModern.tsx]: Ill submit a PR soon hopefully.

**Pull request merged**